src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncCLI.scala

```diff
@@ -0,0 +1,171 @@
+// Copyright 2019 Wikimedia Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.wikimedia.analytics.hdfstools
+
+import java.net.URI
+
+import org.apache.log4j.Level
+import scopt.OptionParser
+
+object HdfsRsyncCLI {
+
+    /**
+     * Command line options parser
+     */
+    val argsParser = new OptionParser[HdfsRsyncConfig]("HDFS-Rsync") {
+        head("HDFS-Rsync Tool", "Provides rsync equivalent to copy data between HDFS and local FS.")
+
+        note(
+            """
+              |To prevent mistakes between local and remote filesystems, src and dst have to be provided as
+              |fully qualified absolute URIs, for instance file:/home or hdfs:///user/hive.
+              |
+              |src should be a glob leading to some existing files and dst should be an existing folder.
+              |Note: a trailing slash in src (as in /example/src/) is changed to a pattern matching only the
+              |      directory content (as in /example/src/*) mimicing linux rsync behavior.
+              |
+              |Syntax for filter/include/exclude rules is similar to the linux rsync one:
```

Comment:
rsync works like this on more than just linux, maybe we can call the 'unix rsync' or maybe the 'GNU rsync'? (it isn't a GNU tool, but it does have a GNU license).  Maybe 'standard rsync' is good?

Reply:
Done